Makefile

```diff
@@ -1,9 +1,16 @@
 # TODO: this is a bit minimalistic isn't it?
 
+USE_SYSTEMD = 1
```

Comment:
I suggest `USE_SYSTEMD ?= 1`

That would allow `USE_SYSTEMD=0 make` in addition to `make USE_SYSTEMD=0`

Reply:
Sounds good! Will do that ^^